### ISSUE TITLE
Option to disable live wallpaper?

### ISSUE THREAD

**Issue by vinoff** [WangDaYeeeeee/GeometricWeather#349]
I am not very keen on animations, I prefer static images. Is there an option to disable the live wallpaper? I couldn't find any. If there isn't, please consider this a feature request!

Thank you for the awesome app.

I am using F-Droid's version btw.

**Comment by mariobrostech** [comment]
I would like for this to be an option as well. Not only is it good for those who prefer to see static images to save battery, but it would also help with accessibility for people who are sensitive to motion.

**Comment by WangDaYeeeeee** [comment]
You mean the weather animation inside app? or live wallpaper displayed in launcher?

**Comment by mariobrostech** [comment]
I’m referring to the weather animation in the app. Similar how you can turn off the gyroscope sensors, there should be a way to disable the weather animations shown in the app as well.

**Comment by jgottfried** [comment]
I think an option to disable the animations (both in app background and in the live wallpaper) would be nice.
I like a dynamic wallpaper in my launcher that changes with current weather conditions, but some of the animations are a bit too fast and "busy" for my personal taste. Also my phone is old and I worry about resource/battery usage.
So it would be neat to have a static version of those.

**Comment by glocalglocal** [comment]
I absolutely hate that animation. Completely unnecessary, keeps my old phone busy, and annoying, plus I suspect it may have something to do with #435 which I just reported. I am guessing that animation alone must keep some users who are less tolerant to animation away from the app.

**Comment by papjul** [comment]
I agree, Geometric Weather should at least honor the "disable motion" accessibility option from Android settings.
